### ISSUE TITLE
testing glucose example

### ISSUE THREAD

**Issue by pgreisen** [mcs07/PubChemPy#28]
Hi , 
I am trying to test the glucose example to retrieve compunds from pubchem - I am using a jupyter notebook with python 3:

import pubchempy as pcp
results = pcp.get_compounds('Glucose', 'name')
results

which only returns one of the 

5793
C([C@@H]1[C@H]([C@@H]([C@H](C(O1)O)O)O)O)O

where I would expect the different forms of glc - the type is a list. has there been any changes to this lately ?

**Comment by mcs07** [comment]
Yes, PubChem now manually override the search results for some common chemical chemical names. There is a section about this in the docs: [Getting a full results list for common compound names](http://pubchempy.readthedocs.io/en/latest/guide/searching.html#getting-a-full-results-list-for-common-compound-names)

I should replace the glucose search example in the "Getting Started" section with some other compound that returns multiple results, to make things clearer.
